File: src/src_platform_uwp/fmg/uwp/draw/img/StaticRotateImgConsts.cs
using Windows.UI.Core;
using fmg.core.img;
using fmg.uwp.utils;

namespace fmg.uwp.draw.img {

   static class StaticRotateImgConsts {

      static StaticRotateImgConsts() {
         ImageModelConsts.DeferrInvoker = doRun => AsyncRunner.InvokeFromUiLater(() => doRun(), CoreDispatcherPriority.Normal);
         RotatedImgConst.TimerCreator = () => new Timer();
      }

      public static void Init() {
         // implicit call static block
      }

   }

}

File: src/src_platform_jfx/fmg/jfx/draw/img/StaticRotateImgConsts.java
package fmg.jfx.draw.img;

import fmg.core.img.RotatedImg;
import fmg.core.img.StaticImg;
import javafx.application.Platform;

final class StaticRotateImgConsts {

   static {
      StaticImg.DEFERR_INVOKER = doRun -> Platform.runLater(doRun);
      RotatedImg.TIMER_CREATOR = () -> new fmg.jfx.utils.Timer();
   }

   public static void init() {
      // implicit call static block
   }

}
